ui/app/AppLayouts/Chat/views/communities/CommunityNewAirdropView.qml

```diff
@@ -105,6 +196,55 @@ StatusScrollView {
                 accountName: modelItem.accountName
             }
         }
+
+        function rebuildFeesModel() {
+            feesModel.clear()
+
+            const airdropTokens = ModelUtils.modelToArray(
+                                    root.selectedHoldingsModel,
+                                    ["contractUniqueKey", "accountName",
```

Comment:
Are `contractUniqueKey` and `key` redundant? I guess we should need only `key` to identify the item, where the **key** is in this case the **contract unique identifier** data, and then keep the `symbol` information as `symbol` not as `key`.

```suggestion
                                    ["key", "accountName","symbol", "amount", "tokenText",
                                     "networkText"]
```

Reply:
Good point, I improved that a bit, but probably it's the subject for discussion with the backend guys at some point.

Reply:
Thank you! Next week we can meet and talk about that bc I think we need to unify things!!